NoteFind/View Controllers/FeedViewController.m

```diff
@@ -35,7 +44,23 @@ - (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
 }
  */
 
+<<<<<<< Updated upstream
 - (IBAction)didTapAdd:(id)sender {
+=======
+- (void) getNotes {
+    PFQuery *noteQuery = [Note query];
+    [noteQuery orderByDescending:@"createdAt"];
+    [noteQuery includeKey:@"author"];
+    
+    [noteQuery findObjectsInBackgroundWithBlock:^(NSArray<Note *> *notes, NSError *error) {
+        if (notes) {
+            self.notes = (NSMutableArray *)notes;
```

Comment:
Generally in Objective-C Blocks, it's discouraged to use `self` directly since the Block will implicitly retain it. If `self` retains the block then you have a retain cycle and likely memory leak over time.

the general accepted pattern is to create a `weak` reference to `self` and then within the scope of the block assign that to a `strongly` held variable that is used. https://medium.com/fantageek/understanding-weak-and-strong-in-objective-c-d17ba4c2c297